lefthook.yml

```
@@ -0,0 +1,47 @@
+pre-commit:
+  parallel: true
+  jobs:
+    - name: Format sidebars
+      glob: "files/sidebars/*.yaml"
+      run: yarn content fmt-sidebars
+
+    - name: Check URL issues
+      glob: "*"
+      run: node scripts/log-url-issues.js
+
+    - name: Fix redirects
+      glob: "files/en-us/_redirects.txt"
+      run: yarn content fix-redirects en-US && yarn content validate-redirects en-US
+
+    - name: Format non-md
+      glob: "*.{js,json,yml,yaml,css,html}"
+      run: yarn prettier --ignore-unknown --write {staged_files}
+
+    - name: Run 'yarn filecheck'
+      glob: "*.{svg,png,jpeg,jpg,gif}"
+      run: yarn filecheck {staged_files}
+
+    - name: Fix markdown
+      glob: "*.md"
+      group:
+        parallel: true
+        jobs:
+          - name: Front matter
+            run: node scripts/front-matter_linter.js --fix true {staged_files}
+          - name: Markdownlint
+            run: yarn markdownlint-cli2 --fix {staged_files}
```

```suggestion
            run: yarn markdownlint-cli2 --fix {staged_files}
            stage_fixed: true
```
All the `run:`s in the file need `stage_fixed: true`. Without the `stage_fixed` the fixed changes don't get committed. It will be annoying to ask contributors to make a second commit to get lint fixes in.

---

done in https://github.com/mdn/content/pull/38399/commits/4c31e53fe5d2e6ba4993588ea55d331c878e4c79